src/sentry/web/urls.py

```diff
@@ -366,7 +369,7 @@ def init_all_applications():
     # Keep named URL for for things using reverse
     url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/issues/(?P<group_id>\d+)/$', react_page_view,
         name='sentry-group'),
-    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/$', react_page_view,
+    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/$', ReactProjectPageView.as_view(),
```

Comment:
This broke some of the email tests since `project_id` changed to `project_slug`, so either this needs to go back to `project_id`, or tests need updated to reflect this.